.travis.yml

```diff
@@ -105,6 +122,18 @@ jobs:
         on:
           tags: true
 
+stages:
+  - name: check
```

Comment:
`stages:` declaration is better to be placed on tob, before `jobs:` to provide an approximate overview of what's going on.